cdk/lib/__snapshots__/commercial-canaries.test.ts.snap

```diff
@@ -1686,7 +1686,7 @@ Object {
 }
 `;
 
-exports[`The CommercialCanaries stack matches the US CODE snapshot 1`] = `
+exports[`The CommercialCanaries stack matches the USA CODE snapshot 1`] = `
```

Comment:
I do understand the reason behind changing `UK` to `Europe` but is it necessary to change this in here? I always hear everyone at the Guardian say `US` unless you want to follow the regions location and not the location abbreviation 🤔